library/spit_results.py

```diff
@@ -39,10 +199,17 @@ def write_to_csv(self):
             file_size = os.path.getsize(f_path)
             vals = self.vals
             fields = sorted(vals[0].keys())
+            try:
+                fields.remove('systemid.contents')
```

Comment:
There might be a problem if `redhat-packages.results` is in fields but `systemid.contents` is not.

I'll send a PR today which will turn fields into a set, which should make this much easier to solve. Without that I don't really see an elegant way to do it.